cli.js

```diff
@@ -49,13 +49,11 @@ if(program.args.join().toUpperCase().indexOf('::') !== -1) {
     logUpdate.clear();
     compareInfo(movies) 
   });
-  }
-
-else {
-  var interval = setInterval(function() {
+  } else {
+  const interval = setInterval(function() {
   logUpdate("Loading..." + chalk.cyan.bold.dim(frame()));
   }, 50)
-  fetch('http://www.omdbapi.com/?t='+ program.args.join().trim().replace(/ /g,"+")+'&tomatoes=true')
+  fetch('http://www.omdbapi.com/?apikey=[yourkey]&t='+ program.args.join().trim().replace(/ /g,"+")')
```

Comment:
It looks like you left an extra single quote on the end?

Does node support template strings? If so, I would do:

```js
const baseUrl = `http://www.omdbapi.com/?apikey=${yourkey}`;
const title = program.args.join().trim().replace(/ /g,"+");
fetch(`${baseUrl}&t=${title}`);
```

Reply:
And then it would probably be good to get the API key from a config file somewhere on the user's computer.